content/millennium/dstu2/individuals/patient.md

```diff
@@ -86,12 +107,28 @@ _Implementation Notes_
 
 * The [Patient.animal] modifier element is not supported and will not be returned.
 
-### Response
+### Authorization Types
+
+<%= authorization_types(practitioner: true, patient: true, system: true)%>
+
+### Headers
+
+To successfully GET a Patient, the following headers must be provided. Authorization must be provided if accessing the closed endpoint.
+
+        Accept-Type: application/json+fhir
+        Authorization: <OAuth2 Bearer Token>
+
+### Example
+#### Request
```

Comment:
Looks like this displays as hashes on the site, I think because there is no blank line between 121 and 122